src/web/js/trove/chart-lib.js

```diff
@@ -530,21 +532,34 @@
           // The only way to hijack marker events is to walk the DOM here
           // If Google changes the DOM, these lines will likely break
           const svgRoot = chart.container.querySelector('svg');
-          const slices = [...svgRoot.children].slice(2, -1);
+
+          // The order of SVG slices is *not* the order of the rows in the table!!
+          // 2 slices: drawn in reverse order
+          // >2 slices: the first row in the table is the 
```

Comment:
Sounds like the comment is incomplete.

Reply:
@schanzer I meant "is the" seems to abruptly end. But it's great that this is now more exhaustive as well!

Reply:
Oh! I didn't even noticed that got cut off. Fixed in the latest commit